Ports/.port_include.sh

```diff
@@ -315,6 +315,11 @@ fetch_simple() {
     filename="${2}"
     checksum="${3}"
 
+    if [ "${filename}" != "$(basename "${url}")" ]; then
```

Comment:
uhhh...? servers are very much allowed to set a filename, and URLs are allowed to have literal garbage as their "basename", this seems meaningless to me.

Reply:
The server is not involved at this point, we write the output to a file manually (which should fully ignore that particular `Content-Disposition`). As for garbage basenames, that I answered below.

This is primarily just a check that helped me find all ports that didn't already have their filename and URL matching, so I could figure out whether any of them actually depend on that particular name.